elfo-test/src/proxy.rs

```diff
@@ -80,6 +81,15 @@ impl Proxy {
         })
     }
 
+    pub fn try_request<R: Request>(
```

Comment:
Why is it an async method? Proxy methods should mirror context ones. But `Context::try_request` will be sync like `try_send`